src/Sylius/Behat/Page/Shop/Checkout/CompletePage.php

```diff
@@ -291,6 +297,25 @@ public function getShippingPromotionDiscount($promotionName)
     /**
      * {@inheritdoc}
      */
+    public function tryToOpen(array $urlParameters = [])
+    {
+        if ($this->getDriver() instanceof Selenium2Driver) {
+            $start = microtime(true);
+            $end = $start + 15;
+            do {
+                parent::tryToOpen($urlParameters);
+                sleep(1);
+            } while(!$this->isOpen() && microtime(true) < $end);
```

Comment:
So if it takes over 1s to load that page, it will repeat it 15 times and always fail instead of waiting a second or two more to get the page loaded.